frontend/src/index.tsx

```diff
@@ -1,39 +1 @@
-import React from 'react';
```

Comment:
I'm sorry I don't understand this, why using bootstrap here and removing everything from index.tsx, I'm assuming there should be a requirement in module ferderation that the entrypoint should be bootstrap or smth like this, but it's not clear, can we add a comment somewhere to leave some context?

Reply:
So having the single file like this for some reason is necessary. I can add a comment.

As for the SWC error, that's new to me. I'll rebase and look into it.